src/wp-includes/update.php

```diff
@@ -138,6 +138,10 @@ function wp_version_check( $extra_stats = array(), $force_check = false ) {
 		);
 	}
 
+	if ( class_exists( 'SQLite3' ) ) {
+		$query['sqlite'] = SQLite3::version()['versionString'];
```

Comment:
This should be combined into the `extensions` key instead.

```suggestion
		// Overwrite the sqlite3 extension version with the actual version.
		$query['extensions']['sqlite3'] = SQLite3::version()['versionString'];
```

`phpversion( 'sqlite3' );` is supposed to return the version of the extension / library, but it seems for sqlite3 it doesn't.

Reply:
Makes sense, thank you!